python/plugins/processing/algs/saga/SagaAlgorithmProvider.py

```diff
@@ -91,34 +81,28 @@ def _loadAlgorithms(self):
             ProcessingLog.addToLog(ProcessingLog.LOG_ERROR,
                                    self.tr('Problem with SAGA installation: SAGA was not found or is not correctly installed'))
             return
-        if version not in self.supportedVersions:
-            lastVersion = sorted(self.supportedVersions.keys())[-1]
-            if version > lastVersion:
-                version = lastVersion
-            else:
-                ProcessingLog.addToLog(ProcessingLog.LOG_ERROR,
-                                       self.tr('Problem with SAGA installation: installed SAGA version (%s) is not supported' % version))
-                return
+
+        if version not in ['2.3.0', '2.3.1']:
```

Comment:
@ghtmtt @saberraz @johanvdw I'm confused then... is then 2.3.2 the version we will have to ask to be packaged (i.e. osgeo4w and macos) and that is expected in Linux repos?